changelog.md

```diff
@@ -22,6 +22,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 
 - Move ELK parameters from ElkLayout to DefautConfiguration.
+- Move addComponent from PluginData to Plugin and update it to drop new component in parent container.
```

Comment:
`Move addComponent from PluginData to Plugin` is confusing to me, since I dont see the removal in PluginData, I just see the addition of `addComponent` in DefaultPlugin

Reply:
Done